[[ SB Inclusions ]] Update standalone builder GUI with new inclusions pane

livecodeali opened the pull request:
Standalone Settings GUI diffs: https://gist.github.com/livecodeali/72ca5d37026a365a09bc011088f718c0

New Inclusions pane:
![screen shot 2016-05-10 at 11 03 25](https://cloud.githubusercontent.com/assets/3985854/15144287/57fe0ee2-16a6-11e6-9292-c4dee05870d8.png)

General pane before:
![screen shot 2016-05-10 at 11 01 48](https://cloud.githubusercontent.com/assets/3985854/15144239/13567284-16a6-11e6-8102-a2f88f20e24e.png)

General pane after:
![screen shot 2016-05-10 at 11 00 37](https://cloud.githubusercontent.com/assets/3985854/15144237/0edf4d52-16a6-11e6-8f8a-c79c98e2bf63.png)

Copy Files pane before:
![screen shot 2016-05-10 at 11 01 58](https://cloud.githubusercontent.com/assets/3985854/15144250/273f14ea-16a6-11e6-8347-e35950cd520b.png)

Copy Files pane after:
![screen shot 2016-05-10 at 11 01 55](https://cloud.githubusercontent.com/assets/3985854/15144253/2ec9d39e-16a6-11e6-862a-68164e6158bc.png)

iOS pane before:
![screen shot 2016-05-10 at 11 03 43](https://cloud.githubusercontent.com/assets/3985854/15144262/3a9c3036-16a6-11e6-9aa2-91e987db6c9a.png)

iOS pane after:
![screen shot 2016-05-10 at 11 03 36](https://cloud.githubusercontent.com/assets/3985854/15144270/409fcaec-16a6-11e6-90cc-5d37723aae61.png)

Android pane before:
![screen shot 2016-05-10 at 11 03 52](https://cloud.githubusercontent.com/assets/3985854/15144276/498fb1ee-16a6-11e6-8c42-09c9faf04709.png)

Android pane after:
![screen shot 2016-05-10 at 11 03 48](https://cloud.githubusercontent.com/assets/3985854/15144281/4fb3f18e-16a6-11e6-93ad-1eaee6b9fef5.png)

Commit:
[[ SB Inclusions ]] Move fixed list of externals and DB drivers into IDE Library

Commit:
[[ SB Inclusions ]] Update standalone settings stack
- Remove inclusions lists from General pane
- Remove extensions lists from Copy Files pane
- Add new Inclusions pane
- Add DataView showing available inclusions' data
- Update standalone settings custom prop with selected inclusions
- Remove inclusion options from Android and iOS panes

livecodeali commented:
Requires https://github.com/livecode/livecode/pull/3999

livecodepanos commented:
Just a note regarding the new "Android" pane. The "In App Purchasing" section should expand if you tick the checkbox. And if you select "Samsung", it expands even more. So we have to make sure that after the expansion no content is chopped off.

Commit:
[[ SB Inclusions ]] Tweak android card to show all possible items

livecodeali commented:
Thanks @livecodepanos, well spotted. Here's an improved version
![screen shot 2016-05-11 at 14 18 49](https://cloud.githubusercontent.com/assets/3985854/15182367/fce3ca2c-1783-11e6-989f-2b751d3ac5ba.png)

Commit:
[[ SB Inclusions ]] Add release note

Commit:
[[ Standalone Builder ]] Make sure intended build restrictions are honoured
- All of the desktop platforms can be built at the same time, but no others
- Both the mobile platforms can be built at the same time, but no others
- Emscripten must be built on its own

Removing these restrictions seems to cause problems with the opening and closing
of user stacks.

Commit:
[[ Bug 17189 ]] Replace 'Revolution' with 'LiveCode'

peter-b commented on notes/feature-standalone_inclusions_ui.md:
```diff
@@ -0,0 +1,12 @@
+# Standalone Inclusions Interface
+
+The standalone settings user interface has been reworked to unify the 
+notion of app inclusion. There is now an 'Inlcusions' pane which allows
```
"Inlcusions" -> "Inclusions"

peter-b commented:
@livecode-vulcan review ok 134e866

livecode-vulcan commented:
💙 review by @peter-b ok 134e866fb9f064cd4e2aa5b5deb29524c579afec
